src/stratigraphy/util/interval.py

```diff
@@ -50,6 +51,19 @@ def to_json(self):
         }
 
 
+@dataclass
+class AnnotatedInterval:
+    """Class for annotated intervals."""
+
+    start: float
+    end: float
+    background_rect: fitz.Rect
+
+    @property
+    def line_anchor(self) -> fitz.Point:
+        return fitz.Point(0, 0)
```

Comment:
What's the point of this?

Reply:
That's a good question. I need to check myself. I believe the property was required, but we don't need it for AnnotatedInterval. I would tend to drop it and see if we really need line_anchor for the `AnnotatedInterval` class.

Reply:
Method is removed. Thank you for spotting it.